projects/samples/robotbenchmark/robot_programming/worlds/.robot_programming.wbproj

```diff
@@ -6,6 +6,9 @@ maximizedDockId: -1
 centralWidgetVisible: 1
 selectionDisabled: 0
 viewpointLocked: 0
+3dContextMenuDisabled: 1
+objectMoveDisabled: 1
+forceAndTorqueDisabled: 1
```

Comment:
Yes, this is another option.

For the moment I made these lines optional, so that only few WBPROJ will contain them, and I kept the original formatting to avoid having to support two different formats. Given that in terms of number of characters, the options to have the user interactions on multiple or a single line  are equivalent, it seemed more human-readable and human-writable to have them on multiple lines.

If you really think that changing the format will bring some advantages I can change it.

Reply:
I don't think this is really important, this was just a suggestion.
For me the main advantage would be that it simplifies and make a bit more generic the code.
E.g.
```
  if (mDisabledUserInteractionsMap.contains(WbAction::DISABLE_SELECTION))
    out << "selectionDisabled: " << (int)mDisabledUserInteractionsMap.value(WbAction::DISABLE_SELECTION, false) << "\n";
  if (mDisabledUserInteractionsMap.contains(WbAction::LOCK_VIEWPOINT))
    out << "viewpointLocked: " << (int)mDisabledUserInteractionsMap.value(WbAction::LOCK_VIEWPOINT, false) << "\n";
  if (mDisabledUserInteractionsMap.contains(WbAction::DISABLE_3D_VIEW_CONTEXT_MENU))
    out << "3dContextMenuDisabled: " << (int)mDisabledUserInteractionsMap.value(WbAction::DISABLE_3D_VIEW_CONTEXT_MENU, false)
        << "\n";
  if (mDisabledUserInteractionsMap.contains(WbAction::DISABLE_OBJECT_MOVE))
    out << "objectMoveDisabled: " << (int)mDisabledUserInteractionsMap.value(WbAction::DISABLE_OBJECT_MOVE, false) << "\n";
  if (mDisabledUserInteractionsMap.contains(WbAction::DISABLE_FORCE_AND_TORQUE))
    out << "forceAndTorqueDisabled: " << (int)mDisabledUserInteractionsMap.value(WbAction::DISABLE_FORCE_AND_TORQUE, false)
        << "\n";
  if (mDisabledUserInteractionsMap.contains(WbAction::DISABLE_FAST_MODE))
    out << "fastModeDisabled: " << (int)mDisabledUserInteractionsMap.value(WbAction::DISABLE_FAST_MODE, false) << "\n";
```
VS
```
if (!mEnabledOptionalRenderingList.isEmpty())
    out << "globalOptionalRendering: " << joinUniqueNameList(mEnabledOptionalRenderingList) << "\n";
```

But that's fine like it is now too for me.

Reply:
Ok. I can apply it and increase the WBPROJ version number.